pkgs/native_toolchain_c/lib/src/native_toolchain/gcc.dart

```diff
@@ -29,6 +29,15 @@ final i686LinuxGnuGccAr = _gnuArchiver('i686-linux-gnu');
 /// [gnuLinker] with [Tool.defaultResolver] for [Architecture.ia32].
 final i686LinuxGnuLd = _gnuLinker('i686-linux-gnu');
 
+/// [gcc] with [Tool.defaultResolver] for [Architecture.x64].
```

Comment:
Please add a comment to `gcc`, `gnuArchiver`, and `gnuLinker` above that it is for `[Architecture.current]`.

Reply:
Done.